mono/metadata/w32file-unix.c

```diff
@@ -3473,8 +3473,6 @@ mono_w32file_find_first (const gunichar2 *pattern, WIN32_FIND_DATA *find_data)
 	if (handle == INVALID_HANDLE_VALUE) {
 		g_warning ("%s: error creating find handle", __func__);
 		g_free (dir_part);
-		g_free (entry_part);
-		g_free (utf8_pattern);
```

Comment:
These were taken care of on line 3463 and 3464, right?

Reply:
Yes, we had double free of the same memory here